docs/source/format/Columnar.rst

```diff
@@ -106,8 +106,10 @@ the different physical layouts defined by Arrow:
 * **Primitive (fixed-size)**: a sequence of values each having the
   same byte or bit width
 * **Variable-size Binary**: a sequence of values each having a variable
-  byte length. Two variants of this layout are supported using 32-bit
-  and 64-bit length encoding.
+  byte length. Three variants of this layout are supported using
+  * 32-bit offset encoding
+  * 64-bit offset encoding
+  * 128-bit view-or-inline encoding
```

Comment:
I can see both sides of the argument. From the standpoint of "it encodes the same type of data" (a variable-size binary value) then including it in this list makes sense, and is perhaps somewhat less confusing for a reader of the specification.

Reply:
As in: a separate bullet from `Variable-size Binary`? If so I'd agree, though if we do we should make sure to differentiate it by saying something like "Unlike the Variable-size binary layout, the elements in this layout may be out-of-order or even overlapping." Without any clarification on the differences, I agree it might be confusing for a reader of the spec.

Reply:
I'll amend the list with a new bullet point for `Views of Variable-size Binary`